### PR TITLE
More cheerful logging messages

### PR DESCRIPTION
**Pull request by teh3vil**
I've made some adjustments to try and make the bot a bit more friendly to use. I've made a hell of a mess of it, but I think the idea still stands. Will continue working on this. This is less a proper pull request so much as a suggestion by example :) 

---
I never really got that the messages say things like "forts". I mean, i know they -are- as far as the game and code is concerned, but to fans of the game it's better to see the pokemon lingo :) So I've rewritten (and messed up) the info logging to match my personal tastes. Might be worth having a discussion around what the [Status] stuff should say.
Also added an option to hide the heartbeat.
----
Also added reverse lookup of GPS coordinates so it displays pretty names for pokestops/destinations. Very broken. Looks cool though.
----
Beginnings of implementing friendly move names and exporting them along with other data to tidy .csv files. Makes viewing the data much easier in excel.

### CODE REVIEW

**Comment by infinitewarp** [comment]
It looks like you're including compiled pyc files here, but those should not be shared as they can be platform-dependent. Please remove them and verify that your local git client is correctly respecting the `.gitignore` file.

**Comment by teh3vil** [comment]
Sorry! Will do.

File: pgoapi/pgoapi.py [hunk 1]
```diff
@@ -338,18 +373,17 @@ def spin_nearest_fort(self):
...
-            self.log.info('Nearest fort distance is %s', nearest_fort_dis)
+            self.log.info('[STANDING]\t- Nearest PokeStop is %sm away', int(nearest_fort_dis))
 
             # Fort is close enough to change our route and walk to
             if nearest_fort_dis > 40.00 and nearest_fort_dis <= 100:
                 lat = nearest_fort['latitude']
-                long = nearest_fort['longitude']
-                self.walk_to_fort(destinations[0], directly=True)
+                lon = nearest_fort['longitude']
+                self.walk_to_fort(destinations[0][0], directly=False)
```
Comment (rirze) [inline pgoapi/pgoapi.py]: Why is the keyword `directly` changed here from `True` to `False`?

**Comment by rirze** [comment]
First I gotta say I love the new messages, really makes the process more engaging. However I might even say that the new logging messages may be too creative and might be sacrificing clarity in some cases. I'd like to hear other people's opinions about this.

File: pgoapi/pgoapi.py (hunk 1 above)
Comment (teh3vil) [inline pgoapi/pgoapi.py]: I'm a complete noob! I've been messing around with a whole lot of the code just to see what happens. I should probably check through everything before I commit; this is certainly not ready to roll out code!

**Comment by teh3vil** [comment]
@rirze I agree, some of them can be quite unclear (and some have even been removed/commented out).
I just wanted to initiate a discussion really! Feel free to close off this pull request as the code/messages are nowhere close to good enough for a public release.

**Comment by rirze** [comment]
@teh3vil Darn, I really wanted to see this idea fleshed out. Your points about forts and pokemon lingo were very valid. I would like to see even basic or minimal changes to the logging messages! This would be really valuable to the project and I think you should continue working on this ;)

**Comment by teh3vil** [comment]
I've just got a fresh clone of the developer branch downloaded, going to (carefully this time) go through the infos tomorrow on that so i can offer a clean branch to work from :)

**Comment by albertonoys** [comment]
Have you tried coloring the messages? It would be awesome to have different colors for inventory, traveling, catching pokemons...

I've tried several libraries but none of them work as I expected to (or maybe I can't make them work as I want to, this is more likely), or it needs a big refactor in order to do something like this.

**Comment by teh3vil** [comment]
I've commited some changes to the dev branch with a revised version of these info messages and gave up on exporting .csv after I got the web service working.

**Pull request closed without merging**